zap/src/scan.py

```diff
@@ -360,7 +361,11 @@ def clean_uri_path(xml_report):
     # this should remove the hash.
     for uri in root.iter('uri'):
         r=urlparse(uri.text)
-        r=r._replace(path=URI_HASH_REGEX.sub('', r.path))
+        r_prev = r
+        r=r._replace(path=URI_HASH_REGEX1.sub('', r.path))
+        r=r._replace(path=URI_HASH_REGEX2.sub('index-hash-.js', r.path))
```

Comment:
is the only problem index.js?

Reply:
The only one I saw.

Reply:
I'm sorry to say I have pushed a change here to remove the hash as discussed in the meeting.

Reply:
Sorry about the timing but not the change. 

I put the replacement strings together with the regexes. Because URI_HASH_REGEX2 always matches "index-XXXXXXX-.js" it gets replaced with "index.js", so it should be consistent with how the previous regex works -- just removing the hash.